mycpp/testpkg/module2.py

```diff
@@ -11,3 +12,7 @@ def func2():
 
   from testpkg import module1
   log(module1.CONST1)
+
+def func3():
```

Comment:
Even in the test code I think it's better to be consistent about naming, e.g.

DoesCollect() rather than does_collect()

Func3() rather than func3()

Sometimes we follow MyPy naming but here there is no real reason to